@shared/api/internal.ts

```diff
@@ -942,15 +942,15 @@ export const grantAccess = async (url: string): Promise<void> => {
   }
 };
 
-export const handleSignedHwTransaction = async ({
-  signedTransaction,
+export const handleSignedHwPayload = async ({
```

Comment:
since this can now be a transaction OR a signed auth entry, renaming the function to be more indicative of what it actually does